distrepos/mirror_run.py

```diff
@@ -0,0 +1,115 @@
+"""
+This module contains the functions for populating mirrors for a single tag.
+The main entry point is update_mirrors_for_tag(); other functions are helpers.
+"""
+
+import logging
+from distrepos.params import Options, Tag
+from distrepos.tag_run import update_release_repos
+import typing as t
+import socket
+import string
+import os
+import requests
+from datetime import datetime, timedelta
+from pathlib import Path
+
+_log = logging.getLogger(__name__)
+
+def get_baseline_urls() -> t.List[str]:
+    """
+    Get the 
+    """
+    timeout = 5
+    socket.setdefaulttimeout(timeout)
+    if 'repo-itb' in socket.gethostname():
+        return [
+            "http://repo-itb.opensciencegrid.org", 
+            "http://repo-itb.osg-htc.org"
+        ]
+    else:
+        return [
+            "http://repo.opensciencegrid.org", 
+            "http://repo.osg-htc.org"
+        ]
```

Comment:
```suggestion
        return [
            "https://repo-itb.osg-htc.org"
        ]
    else:
        return [
            "https://repo.osg-htc.org"
        ]
```
repo.opensciencegrid.org and repo.osg-htc.org should be aliases so there's no point in having them both there.  Also let's use HTTPS.